packages/cli/src/cmds/validator/keymanager/persistedKeys.ts

```diff
@@ -246,11 +247,32 @@ export class PersistedKeysBackend implements IPersistedKeysBackend {
   }
 }
 
+type KeystoreDecryptOptions = {
+  force?: boolean;
+  onDecrypt?: (index: number, signer: Signer) => void;
+  // Try to use the cache file if it exists
+  cacheFilePath?: string;
+};
+
 export async function decryptKeystoreDefinitions(
   keystoreDefinitions: LocalKeystoreDefinition[],
-  opts: {force?: boolean; onDecrypt?: (index: number, signer: Signer) => void}
+  opts: KeystoreDecryptOptions
 ): Promise<Signer[]> {
-  const signers: Signer[] = [];
+  if (opts.cacheFilePath) {
+    try {
+      const signers = await loadKeystoreCache(opts.cacheFilePath, keystoreDefinitions);
+      if (opts?.onDecrypt) {
+        opts?.onDecrypt(signers.length - 1, signers[signers.length - 1]);
+      }
+      return signers;
+    } catch (e) {
```

Comment:
`e` seems to be unused

if we want to keep it for debugging purposes

```suggestion
    } catch (_e) {
```

or could just completely remove it

```suggestion
    } catch {
```